src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import main.java.duke.command.Command;
+import main.java.duke.exceptions.DukeException;
+import main.java.duke.maincomponents.Parser;
+import main.java.duke.maincomponents.Storage;
+import main.java.duke.maincomponents.TaskList;
+import main.java.duke.maincomponents.Ui;
+
+/**
+ * Main class for Duke that handles the execution of its functions
+ */
 public class Duke {
+    private static final String FILENAME = "dukedata.txt";
+    private static final String FOLDERNAME = "data";
+    private static final String PATH = FOLDERNAME + "/" + FILENAME;
```

Comment:
I think relative file path is prefered

Reply:
yes that would be a better name for the variable